### PR TITLE
bip-0050: Final update to what actually occurred

### PR DESCRIPTION
**Pull request by luke-jr**
@gavinandresen This look okay? Particularly note the addition of the public domain release.

### CODE REVIEW

**Comment by jl2012** [comment]
the name bitcoin core did not exist until 0.9

File: bip-0050.mediawiki [hunk 1]
```diff
@@ -8,27 +8,29 @@
...
 
+With the insufficiently high BDB lock configuration, it implicitly had become a network consensus rule determining block validity (albeit an inconsistent and unsafe rule, since the lock usage could vary from node to node).
+
 Because max-sized blocks had been successfully processed on the testnet, it did not occur to anyone that there could be blocks that were smaller but require more locks than were available. Prior to 0.7 unmodified mining nodes self-imposed a maximum block size of 500,000 bytes, which further prevented this case from being triggered. 0.7 made the target size configurable and miners had been encouraged to increase this target in the week prior to the incident.
 
-Bitcoin 0.8 does not use Berkeley DB. It uses LevelDB instead, which does not require this kind of pre-configuration. Therefore it was able to process the forking block successfully.
+Bitcoin Core 0.8 did not use Berkeley DB. It switched to LevelDB instead, which did not implement the same locking limits as BDB. Therefore it was able to process the forking block successfully.
```
Comment (btcdrak) [inline bip-0050.mediawiki]: Bitcoin "Core" didnt exist until 0.9

**Comment by andre-amorim** [comment]
for some reason I'm here thinking about libgcrypt and the comments "/* (Do not change these values unless synced with the asm code.)  */" it's too deep thoughts, it would take me ages to explain. I'm going in quiet mode, until I have a mathematical formula to express thyself better.

**Comment by afk11** [comment]
@andre-amorim

**Comment by afk11** [comment]
Seriously wonder if andre is a markov bot..

**Pull request merged**